posthog/api/utils.py

```diff
@@ -38,16 +39,21 @@ def get_target_entity(request: request.Request) -> Entity:
         raise ValueError("An entity must be provided for target entity to be determined")
 
 
-def retrieve_entity_from(entity_id: Union[str, int], events: List[Dict], actions: List[Dict]) -> Optional[Dict]:
+def retrieve_entity_from(
```

Comment:
This method is kind of sus. What if the same entity is graphed twice?

Reply:
Yeah i was trying a few different things with this function and the one above `get_target_entity`. What do you think about enforcing and ordering when you pass the list of events/actions to the api and selecting based on the order? Any other method is fairly sus because you can still have multiple entities with the same id/math/type but not properties.

Reply:
We can also solve this later as the behavior already existed. Leaving it up to you! :)